src/hardware/devices/netbiter/argos/restapi/plugin.pm

```diff
@@ -0,0 +1,52 @@
+#
+# Copyright 2023 Centreon (http://www.centreon.com/)
+#
+# Centreon is a full-fledged industry-strength solution that meets
+# the needs in IT infrastructure and application monitoring for
+# service performance.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+package hardware::devices::netbiter::argos::restapi::plugin;
+
+use strict;
+use warnings;
+use base qw(centreon::plugins::script_custom);
+
+sub new {
+    my ($class, %options) = @_;
+    my $self = $class->SUPER::new(package => __PACKAGE__, %options);
+    bless $self, $class;
+
+    $self->{version} = '1.0';
+    $self->{modes} = {
+        'alarms'       => 'hardware::devices::netbiter::argos::restapi::mode::alarms',
+        'discovery'    => 'hardware::devices::netbiter::argos::restapi::mode::discovery',
+        'list-sensors' => 'hardware::devices::netbiter::argos::restapi::mode::listsensors',
+        'sensor-value' => 'hardware::devices::netbiter::argos::restapi::mode::sensorvalue'
```

Comment:
you could use the name: ```sensors```

Reply:
you're right, forgot to rename it after making it "multi-instances compatible"